apps/posts/api/views.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework import (generics, 
+                            viewsets, 
+                            mixins, 
+                            filters)
```

Comment:
```suggestion
from rest_framework import generics, viewsets, mixins, filters
```

Reply:
@gmelodie not sure if I should be asking you this, but do you think I have those same commits because I created this new branch from the ft/models branch. If yes then how do I avoid this next time?

Regarding this:
```
from rest_framework import (generics, 
                            viewsets, 
                            mixins, 
                            filters)
```
Sometimes I put stuff in parenthesis even when the 75 chars per line pep8 recommendation haven't been exceeded because I imagine that I might be importing lots of other stuff as the application scales. What do you think?